parser/healthchecks.go

```diff
@@ -0,0 +1,245 @@
+
+healthchecks:list        list healthchecks for an app
+healthchecks:set         set healthchecks for an app
+healthchecks:unset       unset healthchecks for an app
+
+Use 'deis help [command]' to learn more.
+`
+
+	switch argv[0] {
+	case "healthchecks:list":
+		return healthchecksList(argv)
+	case "healthchecks:set":
+		return healthchecksSet(argv)
+	case "healthchecks:unset":
+		return healthchecksUnset(argv)
+	default:
+		if printHelp(argv, usage) {
+			return nil
+		}
+
+		if argv[0] == "healthchecks" {
+			argv[0] = "healthchecks:list"
+			return healthchecksList(argv)
+		}
+
+		PrintUsage()
+		return nil
+	}
+}
+
+func healthchecksList(argv []string) error {
+	usage := `
+Lists healthchecks for an application.
+
+Usage: deis healthchecks:list [options]
+
+Options:
+  -a --app=<app>
+    the uniquely identifiable name of the application.
+`
+
+	args, err := docopt.Parse(usage, argv, true, "", false, true)
+
+	if err != nil {
+		return err
+	}
+
+	return cmd.HealthchecksList(safeGetValue(args, "--app"))
+}
+
+func healthchecksSet(argv []string) error {
+	usage := `
+Sets healthchecks for an application.
+
+By default, Workflow only checks that the application starts in their Pod. A health check
+may be added by configuring a health check probe for the application. The health checks
+are implemented as Kubernetes Pod Probes. A 'liveness' and a 'readiness' probe can be
+configured, and each probe can be of type 'httpGet' or 'exec' depending on the type of
+probe the Pod requires.
+
+A 'liveness' probe is useful for applications running for long periods of time, eventually
+transitioning to broken states and cannot recover except by restarting them.
+
+Other times, a 'readiness' probe is useful when the Pod is only temporarily unable to
+serve, and will recover on its own. In this case, if a Pod fails its 'readiness' probe,
+the Pod will not be shut down, but rather the Pod will stop receiving incoming requests.
+
+'httpGet' probes are just as it sounds: it performs a HTTP GET operation on the Pod. A
+response code inside the 200-399 range is considered a pass. 'httpGet' probes accept a
+port number to perform the HTTP GET operation on the Pod.
+
+'exec' probes run a command inside the Pod to determine its health. An exit code of zero
+is considered a pass, while a non-zero status code is considered a fail. 'exec' probes
+accept a string of arguments to be run inside the Pod.
+
+'tcpSocket' probes attempt to open a socket in the Pod. The Pod is only considered healthy
+if the check can establish a connection. 'tcpSocket' probes accept a port number to
+perform the socket connection on the Pod.
+
+Usage: deis healthchecks:set <health-type> <probe-type> [options] [--] <args>...
+
+Arguments:
+  <health-type>
+    the healthcheck type, such as 'liveness' or 'readiness'.
+  <probe-type>
+    the healthcheck probe type, such as 'httpGet', 'exec' or 'tcpSocket'.
+  <args>
+    The arguments required for the healthcheck probe.
```

Comment:
This is in reference to the arguments described above, e.g. exec is a string of args, httpGet uses a port and tcpSocket accepts a port